### DISCUSSION TITLE
emv.codes url broken (REF: Lesson 7 Gas: Cheaper Withdraw (continued))

### DISCUSSION THREAD

**Discussion by mrthedude** [Cyfrin/foundry-full-course-cu#329] [General]
I tried to click on the link in the repo for emv.codes but i think it's broken. I attached a screenshot for reference 
![Screenshot 2023-07-13 160320](https://github.com/Cyfrin/foundry-full-course-f23/assets/95384358/312e7e85-9367-47f9-8176-78913b68e184)

**Comment by cromewar** [comment]
Trying to solve this on the code @PatrickAlphaC but don't know where to point the link, a hand please.

**Comment by PatrickAlphaC** [comment]
This has been fixed! Try now?

**Reply by mrthedude** [reply]
all good now ty!
